data/src/main/scala/ch.epfl.scala.index.data/project/ProjectForm.scala

```diff
@@ -15,9 +20,25 @@ case class ProjectForm(
     cliArtifacts: Set[String],
     // documentation
     customScalaDoc: Option[String],
-    documentationLinks: List[(String, String)]
+    documentationLinks: List[(String, String)],
+    beginnerIssuesLabel: Option[String],
+    chatroom: Option[Url]
 ) {
-  def update(project: Project): Project = {
+  def update(project: Project, paths: DataPaths): Project = {
+    val oldBeginnerIssueLabel = project.github.map(_.beginnerIssuesLabel).getOrElse(None)
+    val beginnerIssues = if (beginnerIssuesLabel != oldBeginnerIssueLabel) {
+      implicit val system = ActorSystem()
```

Comment:
You should probably reuse an existing actor system instead of creating your own.